content/blog/guides/2020/2020-03-25-hello-world/index.mdx

```diff
@@ -1,16 +1,173 @@
 ---
-title: Salted Duck
+title: Collaborative UI Development
 date: '2020-05-25T23:36:03.284Z'
 slug: '/salted-duck'
 desription: 'foo bar'
 draft: true
 ---
 
-# Hello World
+![](./collaboration-illustration.png)
 
-This is my first post on my new fake blog! How exciting!
-I'm sure I'll write a lot more interesting things in the future.
-Oh, and here's a great quote from this Wikipedia on
-[salted duck eggs](http://en.wikipedia.org/wiki/Salted_duck_egg).
+## Introduction
 
-> A salted duck egg is a Chinese preserved food product made by soaking duck
+This guide will help **Frontend Engineers** and people from the **UI Community**
+to develop a common understanding of **collaboration** and why this particular
+skill can improve the process of software development and product design in your
+company. I will **not** describe implementation details or how to create the
+next big Design System. But I will provide you the right methods and practices
```

Comment:
idk do we want to capitalize `design system`? it looks a bit German

Reply:
Yeah we shouldn't capitalize 👍